standardfiles/terraform/.vscode/extensions.json

```diff
@@ -1,3 +1,7 @@
 {
-  "recommendations": ["editorconfig.editorconfig", "hashicorp.terraform"]
+  "recommendations": [
+    "editorconfig.editorconfig",
+    "hashicorp.terraform",
+    "DavidAnson.vscode-markdownlint"
```

Comment:
This extension is by the same person who make markdownlint2 so it makes sense to suggest it.